src/acceptance-test/java/org/zalando/nakadi/webservice/SettingsControllerAT.java

```diff
@@ -58,6 +59,8 @@ public void testGetFlooders() throws Exception {
         final EventType eventType = NakadiTestUtils.createEventType();
         blacklist(eventType.getName(), BlacklistService.Type.CONSUMER_ET);
 
+        Thread.sleep(TimeUnit.SECONDS.toMillis(1)); // give time for TreeCache refresh
```

Comment:
I thought @adyach had improved this test on another PR. He implemented some code that waits for a specific condition, instead of adding a generic `sleep`.